V2 docs (#587)
signalfx/splunk-otel-python: RELEASE_NOTES.md

## Environment Variables

| Variable                                         | Default   | Description                                                                                                        |
|--------------------------------------------------|-----------|--------------------------------------------------------------------------------------------------------------------|
| SPLUNK_ACCESS_TOKEN                              |           | Adds token to requests to enable direct ingest (for skipping the collector)                                        |
| SPLUNK_PROFILER_ENABLED                          | false     | Causes the Splunk profiler to start polling at startup                                                             |

> | SPLUNK_TRACE_RESPONSE_HEADER_ENABLED             | true      | Causes a ServerTimingReponsePropagator to be configured if true (injects tracecontext headers into HTTP responses) |

Reviewer: ```suggestion
| SPLUNK_TRACE_RESPONSE_HEADER_ENABLED             | true      | Configures a ServerTimingReponsePropagator if true (injects tracecontext headers into HTTP responses) |
```

> | SPLUNK_PROFILER_CALL_STACK_INTERVAL              | 1000      | Profiling poll interval in milliseconds                                                                            |

Reviewer: ```suggestion
| SPLUNK_PROFILER_CALL_STACK_INTERVAL              | 1000      | Profiling poll interval, in milliseconds                                                                            |
```

| SPLUNK_PROFILER_LOGS_ENDPOINT                    |           | OTel logging endpoint, only if profiling is enabled                                                                |
| SPLUNK_REALM                                     |           | Sets the metrics and traces endpoints by realm (e.g. us1) and sets the protocol to `http/protobuf`                 |
| OTEL_METRICS_EXPORTER                            | otlp      | Sets the metrics exporter                                                                                          |
| OTEL_TRACES_EXPORTER                             | otlp      | Sets the traces exporter                                                                                           |
| OTEL_LOGS_EXPORTER                               | otlp      | Sets the logs exporter                                                                                             |

> | OTEL_EXPERIMENTAL_RESOURCE_DETECTORS             | host      | Causes a host resource detector to be configured to set telemetry attributes                                       |

Reviewer: ```suggestion
| OTEL_EXPERIMENTAL_RESOURCE_DETECTORS             | host      | Configures a host resource detector to set telemetry attributes                                       |
```

> | OTEL_TRACES_SAMPLER                              | always_on | Causes all traces to be exported                                                                                   |

Reviewer: ```suggestion
| OTEL_TRACES_SAMPLER                              | always_on | Exports all traces                                                            |
```

> | OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED | true      | Causes logs to be exported                                                                                         |

Reviewer: ```suggestion
| OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED | true      | Exports all logs                                                                                        |
```
